apps/src/applab/designMode.js

```diff
@@ -553,6 +553,23 @@ designMode.serializeToLevelHtml = function () {
   Applab.levelHtml = serialization;
 };
 
+
+function getUnsafeHtmlReporter(sanitizationTarget) {
+  function reportUnsafeHtml(sanitizationTarget, removed, unsafe, safe) {
```

Comment:
Good point. I forget when it's safe to rely on the variable in the enclosing scope, but it seems safe in this case. done.

I am returning a function as a way of binding `sanitizationTarget` directly to `reportUnsafeHtml`. The alternative is to pass `sanitizationTarget` as an optional param to `sanitizeHtml`, to `warnAboutUnsafeHtml`, and to `reportUnsafeHtml`. Since there are already other optional params I thought it would be nice to avoid this.